annotations/test.schema.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+
+  "type": "object",
+  "properties": {
+    "instance": {
+      "markdownDescription": "The JSON instance to be annotated."
+    },
+    "assertions": {
+      "markdownDescription": "An array of annotations on the `keyword` - instance `location` pair.",
```

Comment:
add "...that are expected to be present in the output"

Reply:
I updated this, but specifically I don't want to use the word "output" because these are specifically not output tests.